CMakeLists.txt

```diff
@@ -645,32 +620,74 @@ INSTALL(FILES ${CMAKE_BINARY_DIR}/freeglut.pc DESTINATION ${CMAKE_INSTALL_LIBDIR
 # TODO: change the library and .pc name when building for GLES,
 # e.g. -lglut-GLES
 
-INCLUDE(CMakePackageConfigHelpers)
-WRITE_BASIC_PACKAGE_VERSION_FILE(
-    "${CMAKE_CURRENT_BINARY_DIR}/FreeGLUT/FreeGLUTConfigVersion.cmake"
-    VERSION ${VERSION}
-    COMPATIBILITY AnyNewerVersion
+# --- INSTALL ---
+
+set(INSTALL_CONFIGDIR "cmake")
+
+IF(FREEGLUT_BUILD_SHARED_LIBS)
+    install(TARGETS freeglut 
+    EXPORT FreeGLUT-targets
+    RUNTIME DESTINATION "bin"
+    ARCHIVE DESTINATION "lib"
+    LIBRARY DESTINATION "lib"
+    )
+
+    IF(INSTALL_PDB)
+        IF(CMAKE_GENERATOR MATCHES "^Visual Studio" OR
+           CMAKE_GENERATOR MATCHES "Ninja Multi-Config")
+            INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/Debug/freeglut${CMAKE_DEBUG_POSTFIX}.pdb
+                DESTINATION bin
+                CONFIGURATIONS Debug
+                COMPONENT Devel)
+        ELSE()
+            INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/freeglut${CMAKE_DEBUG_POSTFIX}.pdb
+                DESTINATION bin
+                CONFIGURATIONS Debug
+                COMPONENT Devel)
+        ENDIF()
+    ENDIF()
+ENDIF()
+
+IF(FREEGLUT_BUILD_STATIC_LIBS)
+    install(TARGETS freeglut_static 
+    EXPORT FreeGLUT-targets
+    ARCHIVE DESTINATION "lib"
+    LIBRARY DESTINATION "lib"
+    )
+ENDIF()
+
+install(DIRECTORY include/ DESTINATION "include")
```

Comment:
What's the significance of splitting out this include install directory from the previous install commands they were in?